databuilder/extractor/es_last_updated_extractor.py

```diff
@@ -10,9 +10,9 @@
 from databuilder.extractor.generic_extractor import GenericExtractor
 
 
-class Neo4jEsLastUpdatedExtractor(GenericExtractor):
+class EsLastUpdatedExtractor(GenericExtractor):
```

Comment:
Please Note: I changed this to be more generic so all datasources can use this.

Reply:
sounds good, cc @allisonsuarez from Lyft side